### DISCUSSION TITLE
Where can I find the documentation for writing custom builders?

### DISCUSSION THREAD

**Discussion by Ventajou** [storybookjs/storybook#15371] [Documentation]
Hi,

We use an in-house build tool based on esbuild and I'd love to see if I can get it to work as a builder for Storybook. But neither the official docs nor the recent blog posts on builders have information on how to write your own. Does anyone know where I can find details on how to get started?

Thanks!

**Comment by shilman** [comment]
[Here's a draft](https://docs.google.com/document/d/1qTRdD8XQmuBZhRvENinv-dxHvIJNkq8U1Q_h4_NC3pE/edit?usp=sharing) that I hope to add to the official docs soon
